che-incubator/che-code
# Move to npm

### What does this PR do?
- move che-code to npm
- align che-code with upstream

### What issues does this PR fix?

https://github.com/eclipse-che/che/issues/23122

### How to test this PR?
- PR jobs should be green
- testing of any use cases is useful as we pick up a lot of commits from the upstream

I found that the che-code launcher logic was definitely affected, so I've tested the following use cases:
- load static webview resources with the editor from the same domain, see https://github.com/che-incubator/che-code/pull/411
- extension secrets should not be lost when workspace restart, see https://github.com/che-incubator/che-code/pull/340
- browser database name should contain a valid DevWorkspace identifier, see https://github.com/che-incubator/che-code/pull/273


### Does this PR contain changes that override default upstream Code-OSS behavior?
